regression/goto-gcc/gcc_builtins4/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
The commit message says "fix __builtin_classify_type", which is quite confusing in light of this:
1) Please add an explanation of what is being fixed/what the problem was before;
2) moving from `CORE` to `KNOWNBUG` seems contradictory with "fix ..."